src/utils/build_importer.py

```diff
@@ -0,0 +1,116 @@
+from logger import Logger
+from pathlib import Path
+import os
+import re
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from yaml import dump
+
+
+def import_build():
+    Logger.info(f"Paste maxroll.gg build guide or planner build here ie https://maxroll.gg/d4/build-guides/minion-necromancer-guide")
+    url = input()
+    if "maxroll.gg/d4/planner" not in url and "maxroll.gg/d4/build-guides/" not in url:
+        Logger.info(f"Invalid url, please use a d4 planner build on MaxRoll.gg")
+        return
+    options = webdriver.ChromeOptions()
```

Comment:
Since the API for all webdrivers is the same, can't we just let the user decide and then use the appropriate driver? So declare an enum, add a new settings that is validated against that enum and then here, create proper options/webdriver object.

Chrome, Edge, Firefox, (Chromium) are the most used right now.